docs/conversion_examples_gallery/behavior/video.rst

```diff
@@ -30,7 +30,3 @@ This interface can handle conversions from avi, mov, mp4, wmv, flv and most FFmp
     >>>
     >>> # Choose a path for saving the nwb file and run the conversion
     >>> interface.run_conversion(nwbfile_path=path_to_save_nwbfile, metadata=metadata)
-    root pynwb.file.NWBFile ...
-    >>> # If the conversion was successful this should evaluate to ``True`` as the file was created.
-    >>> Path(path_to_save_nwbfile).is_file()
-    True
```

Comment:
Might need ellipsis here? I thought that was fixed in Heberto's last PR tho

Reply:
He fixed everywhere except for the video, so I thought I'd remove it here.

Reply:
OK then 👍